docs/tutorials/mastering.md

```diff
@@ -1,60 +1,626 @@
+
+```console
+cat << EOF | sudo tee /etc/netplan/50-sdcore-networks.yaml
+# /etc/netplan/50-sdcore-networks.yaml
+network:
+  ethernets:
+    enp4s0: # Set the correct interface name
+      dhcp4: no # Set yes if dhcp4 assignment is required
+      dhcp6: no # Set yes if dhcp6 assignment is required
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110
+      interfaces:
+        - vlan201
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+      interfaces:
+        - vlan202
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203
+      interfaces:
+        - vlan203
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+      interfaces:
+        - vlan204
+
+  vlans:
+    vlan201:
+      id: 201
+      link: enp4s0
+    vlan202:
+      id: 202
+      link: enp4s0
+    vlan203:
+      id: 203
+      link: enp4s0
+    vlan204:
+      id: 204
+      link: enp4s0
+
+  version: 2
+EOF
+```
+
+```console
+sudo netplan apply
+```
+
+### Install and Configure LXD
+
+Install LXD:
+```console
+sudo snap install lxd
+```
+
+While initializing the LXD, please pay attention to set the size of new loop device and local network bridge. 
+
+- Loop device size should be enough to launch all VMs
+- New lxd bridge should not be created.
+
+Use default settings for other configurations.
+
+```console
+$ lxd init
+Would you like to use LXD clustering? (yes/no) [default=no]: 
+Do you want to configure a new storage pool? (yes/no) [default=yes]: 
+Name of the new storage pool [default=default]: 
+Name of the storage backend to use (dir, lvm, zfs, btrfs, ceph) [default=zfs]: 
+Create a new ZFS pool? (yes/no) [default=yes]: 
+Would you like to use an existing empty block device (e.g. a disk or partition)? (yes/no) [default=no]: 
+Size in GiB of the new loop device (1GiB minimum) [default=30GiB]: 150GiB
+Would you like to connect to a MAAS server? (yes/no) [default=no]: 
+Would you like to create a new local network bridge? (yes/no) [default=yes]: no
+What IPv4 address should be used? (CIDR subnet notation, “auto” or “none”) [default=auto]: 
+What IPv6 address should be used? (CIDR subnet notation, “auto” or “none”) [default=auto]: 
+Would you like the LXD server to be available over the network? (yes/no) [default=no]: 
+Would you like stale cached images to be updated automatically? (yes/no) [default=yes]: 
+Would you like a YAML "lxd init" preseed to be printed? (yes/no) [default=no]: 
```

Comment:
Can't we use a complete lxd init command instead of using the interactive mode?

Reply:
Yes, we configure it by using a preseed.yaml.

Reply:
Done